buildrunner/config.py

```diff
@@ -374,7 +374,7 @@ def load_config(self, cfg_file, ctx=None, log_file=True):
 
         errors = validate_config(**config)
         if errors:
-            raise BuildRunnerConfigurationError(f'Invalid configuration - {errors.count()} errors found:'
+            raise BuildRunnerConfigurationError(f'Invalid configuration: {errors.count()} errors found:'
```

Comment:
What about `Invalid configuration, {errors.count()} error(s) found:`

Reply:
I agree. I like that better. This: looks weird: